main.js

```diff
@@ -43,6 +43,7 @@ module.exports = function(options) {
 		withNavigationHierarchy: false,
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
+		withoutAssetPreloading: false,
```

Comment:
normally we use `with` prefixes……  defaulted to `false` …… althoguh that would be a breaking change i think having asset preloading on by default was probably a bad move in the first place.

n-express is _supposed_ to default to doing as little as possible……